avocado/core/html.py

```diff
@@ -130,20 +130,21 @@ def tests(self):
                    "INTERRUPTED": "danger"}
         test_info = []
         results_dir = self.results_dir(False)
-        for tst in self.result.tests:
-            tst = tst.copy()    # we don't want to override other's results
-            tst["test"] = str(tst["name"])
-            logdir = os.path.join(results_dir, 'test-results', tst['logdir'])
-            tst['logdir'] = os.path.relpath(logdir, self.html_output_dir)
+        for t in self.result.tests:
```

Comment:
well it'd have been better to keep the `tst` name here, but lets see what's further in the code... We can do pep8 fixes later...